src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,30 @@
+package duke;
+
+/**
+ * Represents the user interface of the chatbot
+ */
+public class Ui {
+
+    // Print the farewell message
```

Comment:
comments above methods should be JavaDoc comments, i.e.

```suggestion
    /**
    * Prints the farewell message
    */
```